benchmarks/benchmarks.py

```diff
@@ -0,0 +1,101 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+from arcticdb import Arctic
+from arcticdb.version_store.library import (
+    WritePayload,
+    ReadRequest
+)
+
+import pandas as pd
+import numpy as np
+
+def generate_pseudo_random_dataframe(n, freq="S", end_timestamp="1/1/2023"):
+    """
+    Generates a Data Frame with 2 columns (timestamp and value) and N rows
+    - timestamp contains timestamps with a given frequency that end at end_timestamp
+    - value contains random floats that sum up to approximately N, for easier testing/verifying
+    """
+    # Generate random values such that their sum is equal to N
+    values = np.random.dirichlet(np.ones(n), size=1) * n
+    # Generate timestamps
+    timestamps = pd.date_range(end=end_timestamp, periods=n, freq="S")
+    # Create dataframe
+    df = pd.DataFrame({"timestamp": timestamps, "value": values[0]})
+    return df
+
+def get_prewritten_lib_name(rows):
+    return f"prewritten_{rows}"
+
+class TimeSuite:
+    """
+    An example benchmark that times the performance of various kinds
+    of iterating over dictionaries in Python.
+    """
+    number = 5
+    params = ([1_000, 100_000], [500, 1000])
+    param_names = ['rows', 'num_symbols']
```

Comment:
What do you think about explicitly specifying the `version` here? It's a maintenance burden, but by default the results will get invalidated each time this file changes. So worried we might miss regressions if `version` is left as `None`.